pkg/prometheus/promcfg_test.go

```diff
@@ -12107,7 +12134,18 @@ func TestGenerateAlertmanagerConfig(t *testing.T) {
 				nil,
 				nil,
 				nil,
-				assets.NewTestStoreBuilder(),
+				assets.NewTestStoreBuilder(
+					&v1.Secret{
+						ObjectMeta: metav1.ObjectMeta{
+							Name:      "foo",
+							Namespace: "default",
+						},
+						Data: map[string][]byte{
+							"proxy-header": []byte("value"),
+							"token":        []byte("value"),
```

Comment:
(nit) we don't use this key in the test?
```suggestion
```

Reply:
I have merged the PR since we can include this change as well in release today. 
I can do follow-up PR to remove this